### PR TITLE
Alertmanager integration

### PR DESCRIPTION
**Pull request by Shashankft9**
- adding alerting rules for policy action
- loki installation with retention and alertmanager integration

### CODE REVIEW

**Comment by Shashankft9** [comment]
cc @DelusionalOptimist

File: example/tutorials/tutorial_grafana.md [hunk 1]
```diff
@@ -33,7 +33,13 @@ Depending on your KubeArmor setup, you can follow:
     --set loki.storage.type="filesystem" \
     --set loki.auth_enabled=false \
     --set singleBinary.replicas=1
+
+    ```
+    If integration with Alertmanager is required, along with retention period, we can use `../loki-values.yaml` and run:
+    ```bash
+    helm upgrade --install loki grafana/loki --values ../loki-values.yaml
```
Comment (daemon1024) [inline example/tutorials/tutorial_grafana.md:40]: Can we have a small reference to Alert Manager installation guide so as if people want to install it they can be redirected there?

**Review by daemon1024 — APPROVED** [review]
LGTM, This is nice to have 🙌🏽

**Review by DelusionalOptimist — APPROVED** [review]
Thanks for raising this @Shashankft9. :rocket: 
LGTM.

**Pull request merged**